src/main/java/org/folio/processing/mapping/mapper/mappers/ItemMapper.java

```diff
@@ -50,18 +50,17 @@ private DataImportEventPayload executeMultipleItemsLogic(DataImportEventPayload
     HashMap<String, String> payloadContext = eventPayload.getContext();
     List<MappingRule> mappingRules = profile.getMappingDetails().getMappingFields();
     JsonArray items = new JsonArray();
-    String marcField = payloadContext.get(MULTIPLE_HOLDINGS_FIELD);
-
-    if (marcField == null) {
-      adjustContextToContainEntitiesAsJsonObject(eventPayload, ITEM);
-      writer.initialize(eventPayload);
-      items.add(mapSingleEntity(eventPayload, reader, writer, mappingRules, ITEM.value()));
+    if (isJsonArray(eventPayload.getContext().get(ITEM.value())) && !new JsonArray(eventPayload.getContext().get(ITEM.value())).isEmpty()) {
```

Comment:
correct me if I'm wrong but it seems we could use the defined above `payloadContext`  instead of `eventPayload.getContext()`

Reply:
yes, tnx